index.html

```diff
@@ -46,7 +46,7 @@
       </div>
     </div>
     <!--Band section-->
-    <h2 class="text-center text-4xl" id="band">THE BAND</h2>
```

Comment:
Please use H1 Instead of H2, for better search engine optimisation

Reply:
check now